botocore/parsers.py

```diff
@@ -535,6 +535,13 @@ def _inject_response_metadata(self, parsed, headers):
             parsed.setdefault('ResponseMetadata', {})['RequestId'] = (
                 headers['x-amzn-requestid'])
 
+    def _parse_body_as_json(self, body_contents):
+        if not body_contents:
+            return {}
```

Comment:
This is just me thinking out loud but should there a difference between a body with an empty string and a body with an empty JSON structure? It is really weird in a first place an empty string is the body of a JSON protocol service. I think that there is not a great way to represent it other than an empty dictionary. Have a value of ``None`` is not a good option because there could be no body on the response.